docs/api/specs/preloadDesktopAgents.md

```diff
@@ -0,0 +1,28 @@
+---
+id: preloadDesktopAgents
+sidebar_label: Preload Desktop Agents
+title: Preload Desktop Agents (next)
+---
+
+
+A Preload Desktop Agent is an FDC3 Desktop Agents (DA) supporting web applications that 'injects' or 'preloads' scripts into web windows which provide access to an FDC3 Desktop Agent API implementation. This document specifies the required behavior for a Preload Desktop Agent (DA).
```

Comment:
- I believe "FDC3 Desktop Agents (DA)" should read "FDC3 Desktop Agent (DA)" (singular).
- Perhaps also either change change "'injects' or 'preloads'" to "'inject' or 'preload'"?

Reply:
Fixed in 9f0bc4d0853c0707284b8ff89a1c950f5ff3637e